pkg/node_exporter/Dockerfile

```diff
@@ -1,7 +1,28 @@
-FROM prom/node-exporter:v0.15.1@sha256:88c602650e861bb8045ac83421edecbd056fc58708d47554a0d1a10f19ed7556
+FROM linuxkit/alpine:07f7d136e427dc68154cd5edbb2b9576f9ac5213 as build
+
+RUN apk add --no-cache go git g++ make
+
+ENV GOPATH=/go PATH=$PATH:/go/bin
+ENV GITREPO=github.com/prometheus/node_exporter
+ENV COMMIT=v0.15.0
+
+RUN go get -d ${GITREPO} \
+    && cd /go/src/${GITREPO} \
+    && git checkout ${COMMIT} \
+    && CGO_ENABLED=0 make build \
+    && mv node_exporter /usr/bin/
```

Comment:
minor nit, but here you move it to `/usr/bin` and below it is then copied to `/bin`. Either move int to `/bin` or something like `/out`.